jvb/src/main/java/org/jitsi/videobridge/Main.java

```diff
@@ -155,4 +170,23 @@ public void debug(@NotNull Function0<String> function0)
             }
         });
     }
+
+    private static Server setupPublicHttpServer()
+    {
+        JettyBundleActivatorConfig publicServerConfig = new JettyBundleActivatorConfig(
+            "org.jitsi.videobridge.rest",
+            "videobridge.http-servers.public"
+        );
+
+        final Server publicServer = JettyHelpers.createServer(publicServerConfig);
+        ColibriWebSocketService colibriWebSocketService =
+            new ColibriWebSocketService(publicServerConfig.getKeyStorePath() != null);
```

Comment:
I think the `keyStorePath == null` check is used in 3 places now (here, the new helper, and jicoco). I would push it down to `JettyBundleActivatorConfig`: `val isTls: Boolean = keystorePath != null`

Reply:
We have configuration which enables/disables `ColibriWebSocketService`, which is currently in its constructor. Should we move it here? I.e. only create and register the service if it's enabled?

Reply:
> I think the keyStorePath == null check is used in 3 places now (here, the new helper, and jicoco). I would push it down to JettyBundleActivatorConfig: val isTls: Boolean = keystorePath != null

I think I was avoiding having to mess with Jicoco :/ But yeah I can add it there.

> We have configuration which enables/disables ColibriWebSocketService, which is currently in its constructor. Should we move it here? I.e. only create and register the service if it's enabled?

Yeah I think these are the two approaches for these sorts of services: either always "create" them and let the service itself decide whether or not it's enabled, or have the caller figure it out.  There are aspects of both solutions I don't like:

1. Having the "outside" code decide whether or not to start the service basically requires that the service's config be exposed, which I think it would be nice to avoid.  It also feels like the service itself should know best.

1. Having the service decide requires creating an instance of it (otherwise we have to use an outside config class instead of normal props, which I think is nicer; especially when no outside code _needs_ access to the config).

So here I ended up creating the service, but not registering the servlet if it isn't enabled.